cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -336,6 +340,15 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) caerrors.AutoscalerErr
 		klog.Errorf("Failed to refresh cloud provider config: %v", err)
 		return caerrors.ToAutoscalerError(caerrors.CloudProviderError, err)
 	}
+	if a.loopStartNotifier != nil {
```

Comment:
I'd just assume non-nil. If you want to handle `nil` separately, it doesn't make sense to do the defaulting in `autoscaler.go`.